.github/workflows/support-lambdas-build.yml

```diff
@@ -60,3 +60,68 @@ jobs:
           export LAST_TEAMCITY_BUILD=15000
           export GITHUB_RUN_NUMBER=$(( $GITHUB_RUN_NUMBER + $LAST_TEAMCITY_BUILD ))
           sbt "project support-lambdas" riffRaffUpload
+
+  support_lambdas_typescripts_bigquery_acquisition_publisher:
+    if: >-
```

Comment:
Not sure if we need this if statement any more do we?

Reply:
Hmm good question, I think you may be right. We duplicated this from the other (existing) job in here so I might leave it here and investigat removing both separately.